cloud-governance-mcp/app.py

```
@@ -0,0 +1,770 @@
+        error_msg = f"Error executing {name}: {str(error)}"
+        st.error(error_msg)
+        return error_msg
+
+    # Debug logging - show tool response
+    if debug_mode:
+        with st.expander(f"✅ Tool Response: {name}", expanded=False):
+            try:
+                # Try to pretty-print JSON
+                parsed = json.loads(result)
+                st.code(json.dumps(parsed, indent=2), language="json")
+            except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)